### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by notmarkmiranda**
you're welcome

### CODE REVIEW

**Comment by notmarkmiranda** [comment]
@s-espinosa admin can create demo nights. 

maybe this needs to expand to a unique url or password like casey suggested?

File: app/controllers/admin/demo_nights_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Admin::DemoNightsController < Admin::BaseController
+
+  def new
+    @demo_night = DemoNight.new
+
```
Comment (s-espinosa) [inline app/controllers/admin/demo_nights_controller.rb]: @notmarkmiranda Do we need this blank line here?

File: app/views/admin/demo_nights/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<div class="projects">
```
Comment (s-espinosa) [inline app/views/admin/demo_nights/show.html.erb]: This is an index, right?

File: app/controllers/admin/demo_nights_controller.rb (hunk 1 above)
Comment (notmarkmiranda) [inline app/controllers/admin/demo_nights_controller.rb]: yes.

**Pull request merged**